Applications/WebPortal/WebApp/library/source/components/money_input_box.tsx

```diff
@@ -152,6 +153,48 @@ export class MoneyInputBox extends React.Component<Properties, State> {
     });
   }
 
+  private static readonly STYLE = {
+    input: {
+      boxSizing: 'border-box' as 'border-box',
+      height: '34px',
+      display: 'flex' as 'flex',
+      flexDirection: 'row' as 'row',
+      flexWrap: 'nowrap' as 'nowrap',
+      alignItems: 'center' as 'center',
+      justifyContent: 'space-between',
+      border: '1px solid #C8C8C8',
+      borderRadius: '1px',
+      font: '400 16px Roboto',
+      color: '#000000',
+      flexGrow: 1,
+      maxWidth: '246px',
+      width: '100%',
+      paddingLeft: '10px',
+    }
+  }
```

Comment:
Add semi colons at the end of an expression. So function definitions do not need a semi colon at the end, but when initializing a variable, you do need the semi colon.